INSTALL.md

```diff
@@ -175,6 +189,6 @@ bundle exec rake db:reset # This will drop and setup the database
 ### Run Porta
 Start up the rails server by running the following command:
 ```bash
-$ UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
+$ env UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
```

Comment:
Why the `env`? 🤔

Reply:
@Martouta using fish shell `UNICORN_WORKERS=2` does not work. Their documentation indicates to use `env` http://fishshell.com/docs/current/faq.html#faq-single-env.

And the env works also for bash and zsh. If this is not a problem, I can remove the env 😄

Reply:
Awesome! 👍 🎉